utilitiesApp/src/iocshHelpers.cpp

```diff
@@ -0,0 +1,74 @@
+#include "dbAccessDefs.h"
+#include "dbStaticLib.h"
+#include "epicsString.h"
+#include "envDefs.h"
+#include "iocsh.h"
+#include "epicsExport.h"
+
+/// Search for PVs that match the specified pattern (as dbgrep) and put the number of found PVs in the specified macro.
+///
+/// @code
+///     countdbgrep("NUM_AXES", "*AXIS*")
+/// @endcode 
+///
+/// @param[in] resultvar The name of the macro to output the result into
+/// @param[in] pmask The pattern to search for
+long countdbgrep(const char* resultvar, const char *pmask) {
+    int count = 0;
+    DBENTRY dbentry;
+    DBENTRY *pdbentry = &dbentry;
```

Comment:
my c++ is a bit rusty but can line 18 and 19 be done in one line?  `DBENTRY *pdbentry = new DBENTRY` ?